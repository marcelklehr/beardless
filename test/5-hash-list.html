<ul><li data-template="linklist"><a data-template="linklist.text"></a></li></ul>