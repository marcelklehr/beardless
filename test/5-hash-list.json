{ "linklist":
  [ {"text": "Hello world"}
  , {"text": "It never sleeps"}
  , {"text": "It just idles"}
  ]
}