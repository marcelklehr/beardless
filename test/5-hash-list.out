<ul><li><a>Hello world</a></li><li><a>It never sleeps</a></li><li><a>It just idles</a></li></ul>